app/views/application/_notebook_run.slim

```diff
@@ -1,5 +1,5 @@
 -if @user.member?
   div.ribbon-wrapper
     div.github-fork-ribbon
-      a.run_jupyter href="#{uuid_notebook_path(nb.uuid)}" Run in Jupyter
+      a.run_jupyter href="#{notebook_path(nb.uuid)}" Run in Jupyter
```

Comment:
@mcrutch, This should do it. Good catch. Thought I tested but I suppose not. I lost all of my environments when I accidentally deleted myself a few tickets ago.